snippets/rust/rust.json

```diff
@@ -253,7 +253,7 @@
     "extern-fn": {
         "prefix": "extern-fn",
         "body": [
-            "extern \"C\" fn ${1:name}(${2:arg}: ${3:Type}) -> ${4:RetType} {",
+            "extern \"C\" fn ${1:name}(${2:arg}: ${3:Type}) -> ${4:_} {",
```

Comment:
Could you point me to a doc where it says we should use `_`, because I'm wondering if `${4}` would also accomplish what you're wanting?

Reply:
![image](https://github.com/rafamadriz/friendly-snippets/assets/47633543/45b5f70f-97bb-4749-8ce8-2ef6a217e56b)
![image](https://github.com/rafamadriz/friendly-snippets/assets/47633543/93381351-f332-4669-9946-ebab8364d12d)